src/header/Header.js

```diff
@@ -51,10 +55,16 @@ class Header extends React.Component {
 
   doSearch_() {
     let nextHistory = `/?search=${this.state.search}`;
-    if (!this.state.top500) {
-      nextHistory += '&top500=0';
-    }
+    
+    // clear filters when doing a fresh search
```

Comment:
thanks for this change. you can remove the commented lines, as well as the console log